src/backends/perf_pt/collect.c

```diff
@@ -221,6 +320,14 @@ poll_loop(int perf_fd, int stop_fd, struct perf_event_mmap_page *mmap_hdr,
         {stop_fd,   POLLHUP,            0}
     };
 
+    // Temproary copy space for new samples in the data buffer.
```

Comment:
Typo: "temporary". I've also never heard the term "copy space" before!

Reply:
But you like it ;)

Reply:
Um...